design/RejectedErrorHandling.md

```diff
@@ -0,0 +1,56 @@
+Appendix document for [ErrorHandling.md](ErrorHandling.md).
+
+## Revision 3-ish
+
+The `GPUAdapter` and `GPUDevice` are event targets which receive events about adapter and device status.
+
+```webidl
+partial interface GPUAdapter : EventTarget {
+    readonly attribute boolean isReady;
+};
+
+interface GPUAdapterLostEvent : Event {
+    readonly attribute DOMString reason;
+};
+
+interface GPUAdapterReadyEvent : Event {};
+```
+
+```webidl
+partial interface GPUDevice : EventTarget {};
+
+interface GPUDeviceLostEvent : Event {
+    readonly attribute boolean recoverable;
+    readonly attribute DOMString reason;
+};
+```
+
+If `GPUAdapter`'s `isReady` attribute is false, `createDevice` will fail. 
+`isReady` may be set to `false` when a `"gpudevicelost"` event fires.
+It will always be set to `true` when a `"gpuadapterready"` event fires.
+
+ - `GPUAdapter` `"gpuadapterlost" -> GPUAdapterLostEvent`:
```

Comment:
would it be possible to separate the words with dashes in the reason? i.e. "gpu-adapter-lost"

Reply:
This is a rejected revision, so I don't think it's worth updating.